ts/helpers/attributesHandler.ts

```diff
@@ -9,7 +9,11 @@ export const getAttributeByPath = (slideAttributes, pathArray: string[]) => {
     }
 
     for (const node of pathArray) {
-        slideAttributes = slideAttributes[node] || slideAttributes[0][node];
+        if (Array.isArray(slideAttributes)) {
+            slideAttributes = slideAttributes[node] || slideAttributes[0][node];
+        } else {
+            slideAttributes = slideAttributes[node];
+        }
```

Comment:
not sure about this, so if it is array then you access via `slideAttributes[node]` ... but you do the same if it is not array as well .. so is it that this can be an object or an array and we only need to find if it is an array so we can check index 0?

Reply:
`slideAttributes[node]` is not required if it was an array,  i have removed it. 
I can explain the rest in person regarding the check with array and non array in person if required.